cigar/tools.go

```diff
@@ -1,5 +1,8 @@
 package cigar
 
+// AddCigar ads a cigar struct to the end of a slice, but is smart about checking
```

Comment:
misspelling "ads"

Reply:
Thank you for catching this.  I just pushed a fix